samcli/lib/utils/sam_logging.py

```diff
@@ -28,9 +32,12 @@ def configure_logger(logger, formatter, level):
         if handlers:
             log_stream_handler = handlers[0]
         else:
-            log_stream_handler = logging.StreamHandler()
+            log_stream_handler = (
+                RichHandler(console=Console(stderr=True), show_time=False, show_path=False, show_level=False)
+                if sys.stderr.isatty()
```

Comment:
Why only `stderr`?

Reply:
logging is to stderr on the stream handler.